keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -484,27 +483,18 @@ type CoordinatedLeaseStrategy string
 
 // CoordinatedLeaseStrategy defines the strategy for picking the leader for coordinated leader election.
 const (
-  OldestCompatibilityVersion CoordinatedStrategy = "OldestCompatibilityVersion"
-  NoCoordination CoordinatedStrategy = "NoCoordination"
+  OldestEmulationVersion CoordinatedLeaseStrategy = "OldestEmulationVersion"
 )
 
+// LeaseSpec is a specification of a Lease.
 type LeaseSpec struct {
-	// Strategy indicates the strategy for picking the leader for coordinated leader election
-  // This is filled in from LeaseCandidate.Spec.Strategy or defaulted to NoCoordinationStrategy
-  // if the leader was not elected by the CLE controller.
-	Strategy CoordinatedLeaseStrategy `json:"strategy,omitempty" protobuf:"string,6,opt,name=strategy"`
-
-	// EndofTerm signals to a lease holder that the lease should not be
-	// renewed because a better candidate is available.
-	EndOfTerm bool `json:"endOfTerm,omitempty" protobuf:"boolean,7,opt,name=endOfTerm"`
-
-	// EXISTING FIELDS BELOW
-
 	// holderIdentity contains the identity of the holder of a current lease.
+	// If Coordinated Leader Election is used, the holder identity must be
+	// equal to the elected LeaseCandidate.metadata.name field.
 	// +optional
 	HolderIdentity *string `json:"holderIdentity,omitempty" protobuf:"bytes,1,opt,name=holderIdentity"`
 	// leaseDurationSeconds is a duration that candidates for a lease need
-	// to wait to force acquire it. This is measure against time of last
+	// to wait to force acquire it. This is measured against the time of last
```

Comment:
something with the indention here? Seeing these in red.
<img width="182" alt="Bildschirmfoto 2024-09-09 um 13 38 35" src="https://github.com/user-attachments/assets/95f8ab8f-ae0f-480c-a3d0-1a74f14605d6">